metabrainz/admin/views_test.py

```diff
@@ -18,3 +18,6 @@ def test_statsview_index(self):
 
     def test_statsview_top_ips(self):
         self.assertStatus(self.client.get(url_for('statsview.top_ips')), 302)
+
+    def test_statsview_supporters(self):
+        self.assertStatus(self.client.get(url_for('statsview.supporters')), 302)
```

Comment:
why do we get a 302 here?

Reply:
I don't think the testing the admin pages has been fully setup before and given the limited audience of these pages, I can't find motivation to get the job done.